utils/cards.ts

```diff
@@ -1,23 +1,22 @@
 import * as childProcess from "child_process";
+import * as fs from "fs";
 import * as util from "util";
+import { rmCommand } from "./env";
 
-const isWin = process.platform.startsWith("win");
 const exec = util.promisify(childProcess.exec);
+const stat = util.promisify(fs.stat);
 
-// in *nix `rm` by default skip non-existing files,
-// on win* we have to use another command and check if directory exists
-const rmCommand = isWin
-  ? `if exist "data\\cards" rmdir /s /q "data\\cards"`
-  : "rm -rf data/cards";
-
-export const setupCardsStorage = () =>
-  Promise.resolve()
-    .then(() => exec(rmCommand))
-    .then(() =>
-      exec(
-        "git clone https://github.com/memory-cards/cards data/cards --depth 2"
-      )
+export const setupCardsStorage = async (forceRenew = false) => {
```

Comment:
Minor note: maybe it's also better if we name `isForceRenew` as it's boolean value.

Reply:
done